### PR TITLE
Stored Xss due to Improper Validation  

### PR DESCRIPTION
**Pull request by Tibinsunny**
## ✍️ Description

The latest version of Bludit is affected with a stored XSS.This XSS can be used to run malicious scripts on the 
browser of a second admin as more than one admin can be added to the CMS. 

Initially I identified it as a self XSS, But since multiple level Admins can be  added ,A malicious admin can exploit this vulnerability
to trick or attack a different admin.

## 🕵️‍♂️ Proof of Concept

1:Download Bludit from [Here](https://github.com/bludit/bludit)    
2:Extract the zipfile to an Apache Server    
3:Enter the required Details and Login as Admin
4:Add a New Admin from "Users" section Under Manage . Let the new admin be "X" and the admin is Malicious User
5:Now X can  add a category (both Name and Description)with the XSS payload [` " onfocus="alert(1)" autofocus=" `] 
Note:Add Category is also under Manage Section
6:Now go to Category Section as `First-Admin` and Click on the newly added Category.Now XSS Payload inserted by the Admin X will be fired 

xssnew

## 💥 Impact

This vulnerability would allow a malicious admin to attack another Admin/Admins

## ☎️ Contact

No

## ✅ Checklist

**In my pull request, I have:**

- [x] _Created and populated the `README.md` and `vulnerability.json` files_
- [x] _Provided the repository URL and any applicable [permalinks]([https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files))_
- [x] _Defined all the applicable weaknesses ([CWEs]([https://cwe.mitre.org/](https://cwe.mitre.org/)))_
- [x] _Proposed the CVSS vector items i.e. User Interaction, Attack Complexity_
- [x] _Checked that the vulnerability affects the latest version of the package released_
- [x] _Checked that a fix does not currently exist that remediates this vulnerability_
- [x] _Complied with all applicable laws_

### CODE REVIEW

**Comment by Tibinsunny** [comment]
@Mik317 
Thanks for reviewing 😄😄

File: bounties/other/bludit/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "PR": "",
+        "UI": "",
+        "S": "",
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "3.3"
```
Comment (Mik317) [inline bounties/other/bludit/1/vulnerability.json:35]: I would higher a bit the CVSS score ... even if the user needs to be a `admin`, it's still possible get an impactful behavior :smile:

Screenshot from 2020-08-12 23-24-25


Cheers,
Mik

Comment (Tibinsunny) [reply bounties/other/bludit/1/vulnerability.json]: @Mik317 
Thanks for pointing out :-)  .I have Updated CVSS Score. 😄😄

Regards,
Tibin

**Comment by Tibinsunny** [comment]
@mufeedvh 
 Thanks for the review  😉😉 :-) 

Regards
Tibin

File: bounties/other/bludit/1/vulnerability.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-08-12",
+    "AffectedVersionRange": "*",
+    "Summary": "Cross Site Scripting",
+    "Author": {
```
[the remark this answers is not in the document]
Comment (Tibinsunny) [reply bounties/other/bludit/1/vulnerability.json]: @JamieSlome 
Changed Author Object :-)

Regards
Tibin

[the remark this answers is not in the document]
Comment (JamieSlome) [reply bounties/other/bludit/1/vulnerability.json]: @Tibinsunny - thanks! 🍰

**Pull request merged**